.github/CODEOWNERS

```diff
@@ -775,6 +775,7 @@ packages/kbn-securitysolution-t-grid @elastic/security-detection-engine
 packages/kbn-securitysolution-utils @elastic/security-detection-engine
 packages/kbn-server-http-tools @elastic/kibana-core
 packages/kbn-server-route-repository @elastic/obs-knowledge-team
+packages/kbn-server-route-repository-client @elastic/obs-knowledge-team
```

Comment:
I don't want to block this awesome effort at all, but I'd love if we could socialize this around all the UI teams in observability and then share the ownership of this across all of us, in a way that makes sense...

Reply:
Yes!

I'm slowly working my way to adding Zod support to this library, at which point I will suggest that our guidelines prescribe that all new APIs should use this library to define their routes (since Zod is also the prescribed validation library) to achieve more consistency.

About the codeownership, perhaps at some point it will make sense for the new DevEx working group to have a Github team that could own shared code like this? As I understand, the Knowledge team owns many of these things because it was written by people in the team or because there wasn't a clear place to put it otherwise.

Reply:
PR to add `zod` support https://github.com/elastic/kibana/pull/190244